README.md

```diff
@@ -1,488 +1,57 @@
-```
-
-
-Note2: to find the value for the KUBEADM_JOIN env variable, from the kubernetes master, run:
-`sudo kubeadm token create --ttl 0 --print-join-command` and copy the results.
-Read the k8-readme.md for more info.
-
-
-
-# Kubernates
-The cloudsim server will try to connect to a Kubernates master. To do this, it will look
-for the following config file: `<home>/.kube/config`.
-
-
-For more details and tips about kubernetes, see the `k8-readme.md` file.
-
-
-
-# Linter
-
-1. Get the linter (gometalinter)
-
-    ```
-    cd ~/go_ws
-    ```
+[![Format](https://img.shields.io/github/actions/workflow/status/gazebo-web/cloudsim/format.yaml?branch=main&label=Format)](https://github.com/gazebo-web/cloudsim/actions?query=workflow:"Format")
+[![Test](https://img.shields.io/github/actions/workflow/status/gazebo-web/cloudsim/test.yaml?branch=main&label=Test)](https://github.com/gazebo-web/cloudsim/actions?query=workflow:"Test")
+</div>
 
-    ```
-    curl -L https://git.io/vp6lP | sh
-    ```
+## Getting started
 
-1. Run the linter
+The cloudsim library provides multiple components that are used by applications to run simulations in the cloud. These components usually are an abstraction of a third-party service that will be consumed by cloudsim, and they were created in
+order to let the application decide which cloud provider to use.
 
-    ```
-    ./bin/gometalinter $(go list github.com/gazebo-web/cloudsim/...)
-    ```
+A more extensive description of each component and examples on how to create applications will be introduced in a future version of this document.
 
-Note you can create this bash script:
+### Features
+- A **Simulator** component that allows to schedule simulations in a certain **Platform**.
+- A **Platform** component that groups other components in such a way that represents a certain region in a cloud provider or a custom setup used by applications.
+- A **Machines** component in charge of interacting with a cloud provider and performing various operations (request, termination, count) with instances like _EC2_ machines.
+- A **Storage** component that uploads artifacts produced by a Simulation to a cloud storage like _S3_.
+- A **Cost Calculator** component that allows application to keep track of their expenses and the cost of running simulations in a certain cloud provider.
+- An **Orchestrator** component that interacts with a Kubernetes cluster to launch, stop and restart simulation resources, including but not limited to the following sub-components:
+    - Pods
+    - Nodes
+    - Secrets
+    - Configurations
+    - Network
+    - Ingresses
+    - Services
+- A **Cycler** component that allows cycling over different regions in a cloud provider, implementing different strategies to enable multi-region support.
+- An **Email Sender** that allows notifying users about their simulations.
 
+## Installation
+Using Go CLI
+```shell
+go get github.com/gazebo-web/cloudsim/v4
```

Comment:
Not entirely related to this PR, but the latest tag we have in this repo is `3.6.0`, and that version's `go.mod` file does not contain a package version. 

In another PR, let's please version our `go.mod` file to `v4` and create a new release/tag in this repository so that this command can actually work.

Reply:
After sending my review I have now noticed that there is an open PR (https://github.com/gazebo-web/cloudsim/pull/6) for this already. Resolving.